setup/install/providers/oracle.md

```diff
@@ -5,5 +5,58 @@ sidebar:
   nav: setup
 redirect_from: /setup/providers/oracle/
 ---
+{% include toc %}
+
+In [Oracle Cloud](https://cloud.oracle.com/){:target="\_blank"}, a Spinnaker
+[__Account__](/concepts/providers/#accounts) maps to an [Oracle Cloud Infrastructure user]( https://cloud.oracle.com/en_US/tryit){:target="\_blank"}.
+
+When setting up your Oracle Cloud provider account, you will [use halyard to add
+the account](#add-an-oracle-cloud-account).
+
+## Prerequisites
+
+You will need the followings to enable Oracle Cloud provider in Spinnaker:
+- Create a user in IAM for the person or system who will be using the Spinnaker, and put that user in at 
+least one IAM group with any desired permissions. 
+See [Adding Users](https://docs.cloud.oracle.com/iaas/Content/GSG/Tasks/addingusers.htm){:target="\_blank"}. 
+You can skip this if the user exists already.
+- The user's home region. 
+See [Managing Regions](https://docs.cloud.oracle.com/iaas/Content/Identity/Tasks/managingregions.htm){:target="\_blank"}. 
+(e.g. `--region us-ashburn-1`)
+- RSA key pair in PEM format (minimum 2048 bits).
+See [How to Generate an API Signing Key](https://docs.cloud.oracle.com/iaas/Content/API/Concepts/apisigningkey.htm#How){:target="\_blank"}. 
+(e.g. `--ssh-private-key-file-path /home/ubuntu/.oci/myPrivateKey.pem`)
+- Fingerprint of the public key. 
+See [How to Get the Key's Fingerprint](https://docs.cloud.oracle.com/iaas/Content/API/Concepts/apisigningkey.htm#How3){:target="\_blank"}. 
+(e.g. `--fingerprint 11:22:33:..:aa`)
+- Tenancy's OCID and user's OCID.
+See [Where to Get the Tenancy's OCID and User's OCID](https://docs.cloud.oracle.com/iaas/Content/API/Concepts/apisigningkey.htm#Other){:target="\_blank"}. 
+(e.g. `--tenancyId ocid1.tenancy.oc1..aa... --user-id ocid1.user.oc1..aa...`)
+- Compartment OCID: On Oracle Cloud Console, open the navigation menu. Under Governance and Administration, go to Identity and click Compartments. 
+See [Managing Compartments](https://docs.cloud.oracle.com/iaas/Content/Identity/Tasks/managingcompartments.htm){:target="\_blank"}. 
+(e.g. `--compartment-id ocid1.compartment.oc1..aa...`)
+- Upload the public key from the key pair in the Console. 
```

Comment:
And you can change this last one from "Upload the public key" to "The public key", again making it consistent with the rest of the list, and because your link text includes "Upload," it'll be clear to the user that they need to upload it.

Reply:
I did mean 'upload' the public key to the console. The public key was originally generated 
from the step of "RSA key pair in PEM format" on user's computer as an text file. The user will need to upload/paste the text into the user's settings/profile on the console.

Reply:
Yeah, I understood that. I was merely suggesting the "upload" part was covered by the link out to the how-to-upload docs. But this was merely a nit for consistency of the list. Not  important to change it :)